wit/world.wit

```diff
@@ -24,6 +24,7 @@ world platform {
   import sqlite;
   import key-value;
   import variables;
+  include wasi:observe/imports@0.2.0-draft;
```

Comment:
Is the plan to modify the 2.0 world, or is this a placeholder for dev work?  This is the issue holding up wasi-keyvalue so very interested in the answer..!